Bugzilla/DB/Schema.pm

```diff
@@ -3179,6 +3179,6 @@ database-specific basis:
 
 L<Bugzilla::DB>
 
-L<http://www.bugzilla.org/docs/developer.html#sql-schema>
+L<https://www.bugzilla.org/docs/developer.html#sql-schema>
```

Comment:
Are all www.bugzilla.org//docs/ URLs meant to be re-written to readthedocs?

Reply:
This particular page is not available on readthedocs (yet)